server/controllers/conversation.js

```diff
@@ -0,0 +1,61 @@
+const Conversation = require("../models/Conversation");
+const User = require("../models/User");
+const asyncHandler = require("express-async-handler");
+
+// @route POST /conversations
+// @desc Create new conversation
+//@access Private
+exports.createConversation = asyncHandler(async (req, res, next) => {
+  const { userId } = req.body;
+
+  if (!userId) {
+    res.status(400);
+    throw new Error("UserId param not sent with request");
+  }
+
+  let isConversation = await Conversation.find({
+    isGroupConversation: false,
+    $and: [
+      { users: { $elemMatch: { $eq: req.user.id } } },
+      { users: { $elemMatch: { $eq: userId } } },
+    ],
+  })
+    .populate("users", "-password")
+    .populate("latestMessage");
+
+  isConversation = await User.populate(isConversation, {
+    path: "latestMessage.sender",
+    select: "username  email",
+  });
+
+  if (isConversation.length > 0) {
+    res.send(isConversation[0]);
```

Comment:
I think should send both responses in a consistent manner (e.g. this line and `line 44`).